evap/evaluation/tests/test_commands.py

```diff
@@ -128,7 +128,7 @@ def test_single_result_anonymization(self):
         choices = [choice for choice in CHOICES[question.type].values if choice != NO_ANSWER]
         random.seed(0)
         for answer in choices:
-            count = random.randint(50, 100)
```

Comment:
can we perhaps exclude anonymize.py and test_*.py from the security checks for less `#nosec` (but i'd do that only if it doesn't require another config file)

Reply:
Sadly, the really really great tool codacy does not provide an option to exclude folders / modules / files from their web frontend, so we would need another file.

I guess this resolves this discussion? Please reopen otherwise.